### PR TITLE
docs: Add LICENSE and README files

### PR DESCRIPTION
**Pull request by herrardo**
EX-3909

# Pull Request Template #

Include LICENSE and README file

## Motivation and Context ##

To add clarity to each package we should have always a LICENSE file and an updated README.

## Type of change ##

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## What is the destination branch of this PR? ##

`main`

## How Has This Been Tested? ##

No tests

## Checklist: ##

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by pmareke — APPROVED** [review]
LGTM

File: packages/logger/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+### Usage
+
+```
+import { logger, LogLevel } from '@empathyco/x-logger';
+```
+
+There are different log levels to be set in the console level and server level.
+
+```
+logger.consoleLevel = LogLevel.warn;
```
Comment (tiborux) [inline packages/logger/README.md:22]: Should we add the whole list abouth the different values that you can use??

export enum LogLevel {  silent = 0,  error = 1,  warn = 2,  info = 3,  debug = 4,  trace = 5}

Should we indicate in some way that when you are setting the consoleLevel to warn, you will print lvls 0, 1 and 2. I know that its logic but i can help to the users to know which levels are going to be printed each time that you set the level.

Comment (herrardo) [reply packages/logger/README.md:22]: Go ahead if you can, i think that will be useful

Comment (herrardo) [reply packages/logger/README.md:22]: Done @tiborux

File: packages/logger/README.md [hunk 2]
```diff
@@ -32,3 +38,15 @@ logger.info()
 logger.debug()
 logger.trace()
 ```
+
+Setting the LogLevel to `warn`, for example, will only send to the console if methods of lower levels are called, `logger.error()` and `logger.warn()` in this case.                                                                                                  logger.warn()
```
Comment (tiborux) [inline packages/logger/README.md:42]: Depending on the LogLevel, only some levels will be displayed in the console. Setting the LogLevel to `warn`, the console will display the levels below it and itself, in this case, `error`  and `warn`.

Comment (herrardo) [reply packages/logger/README.md]: Done

**Review by tiborux — APPROVED** [review]

**Review by pmareke — APPROVED** [review]

**Pull request merged**